python/paddle/fluid/tests/unittests/test_conv3d_op.py

```diff
@@ -83,8 +85,7 @@ def setUp(self):
         conv3d_param = {
             'stride': self.stride,
             'pad': self.pad,
-            'dilations': self.dilations,
-            'data_format': 'AnyLayout'  # TODO(dzhwinter) : should be fix latter
+            'dilations': self.dilations
         }
 
         input = np.random.random(self.input_size).astype(self.dtype)
```

Comment:
shouldn't input like `residual_data` also be defined here?

Reply:
The "data_format" need be removed because there is not used by conv3d_forward_naive function.